eo-maven-plugin/src/test/java/org/eolang/maven/BinarizeMojoTest.java

```diff
@@ -46,6 +47,7 @@ final class BinarizeMojoTest {
      * @throws Exception If fails.
      */
     @Test
+    @Disabled
```

Comment:
@volodya-lombrozo This test should be disabled?

Reply:
@Graur No. I just forgot. Thank you!

Reply:
@volodya-lombrozo You can install `rust` on your PC and this test would pass